dashboard/app/models/pd/enrollment_notification.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: pd_enrollment_notifications
+#
+#  id               :integer          not null, primary key
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  pd_enrollment_id :integer          not null
+#  name             :string(255)
+#
+# Indexes
+#
+#  index_pd_enrollment_notifications_on_pd_enrollment_id  (pd_enrollment_id)
+#
+
+class Pd::EnrollmentNotification < ActiveRecord::Base
+  belongs_to :enrollment, class_name: 'Pd::Enrollment', foreign_key: :pd_enrollment_id
+end
```

Comment:
@davidsbailey you can actually configure quite a few of these sorts of naming details in rails.  It's the general layout that is convention-only, and otherwise many patterns follow convention yet have aspects that can be overridden.

In general here, the issue is with namespaces (via Ruby modules, in this case `Pd`), which is a sort of unofficial recommendation without full support in Rails. For example, see this [blog post](http://bclennox.com/namespaces-in-rails-applications) and [this](http://stackoverflow.com/questions/16873972/rails-associations-in-namespaced-models) stackoverflow question. 

I have played around with this and found that it doesn't work other ways. When I just do `belongs_to :pd_enrollment`, accessing it fails with `NameError: uninitialized constant Pd::EnrollmentNotification::PdEnrollment` The class_name field is apparently required here. Either of these work however:

```
belongs_to :pd_enrollment, class_name: 'Pd::Enrollment'
```
or what I have:
```
belongs_to :enrollment, class_name: 'Pd::Enrollment', foreign_key: :pd_enrollment_id
```

I have a slight preference to the latter as within the namespace I don't think we need to re-specify it all over the place. `Pd::EnrollmentNotification.enrollment` is just as clear as `Pd::EnrollmentNotification.pd_enrollment`. Thoughts?